### PR TITLE
Use a curated collection for images & fix caching issues

### PR DESCRIPTION
**Pull request by sanjaypojo**
- Caching issues fixed by varying image resolution marginally for each
  of the 5 images.
- Images now sourced from the “Prestige” collection, enabling hand
  curation
- Updated default image

### CODE REVIEW

File: worker/image-cache.js [hunk 1]
```diff
@@ -11,39 +11,32 @@ imageToBase64Url = function(image) {
...
+  }
+  image.src =  baseUrl + imageResolution[resolutionIndex];
   image.crossOrigin = 'Anonymous';
   image.onload = function() {
     success(image);
   };
 };
 
-updateImage = function(key) {
+updateImage = function(key, resolutionIndex, photoID) {
```
Comment (harimohanraj89) [inline worker/image-cache.js]: Should we add `photoID` now even though we're not using it?

File: worker/image-cache.js [hunk 2]
```diff
@@ -11,39 +11,32 @@ imageToBase64Url = function(image) {
...
+  resolutionIndex = parseInt(resolutionIndex) || 0;
   var image = new Image();
-  var randomCategory = imageCategories[Math.floor(Math.random() * imageCategories.length)];
-  var baseUrl = 'https://source.unsplash.com/category/'
-  image.src =  baseUrl + randomCategory + '/2880x1800';
+  var imageResolution = ['1920x1080', '1920x1090', '1920x1100', '1920x1110', '1920x1120']
+  var baseUrl = 'https://source.unsplash.com/collection/220267/'
+  if (photoID) {
+    baseUrl = 'https://source.unsplash.com/' + photoID + '/'
+  }
```
Comment (harimohanraj89) [inline worker/image-cache.js]: These few lines above could use some `;` love.

File: worker/image-cache.js [hunk 3]
```diff
@@ -11,39 +11,32 @@ imageToBase64Url = function(image) {
   return canvas.toDataURL("image/png");
 };
 
-imageFetch = function(success) {
-  var imageCategories = ["buildings", "nature", "objects", "people", "technology"];
+imageFetch = function(resolutionIndex, photoID, success) {
+  resolutionIndex = parseInt(resolutionIndex) || 0;
```
Comment (harimohanraj89) [inline worker/image-cache.js]: Do we need this? Are there parts of our code that are not sending a value for this? If not, why guard for an undefined value?

File: worker/image-cache.js (hunk 2 above)
Comment (sanjaypojo) [inline worker/image-cache.js]: Good one! Bad habits of coffeescripter :p

File: worker/image-cache.js (hunk 1 above)
Comment (sanjaypojo) [inline worker/image-cache.js]: Yeah, I cupped it n. I initially coded this whole new thing with static images for the initial set of 5. But now that's totally not required. Will remove photoID everywhere.

File: worker/image-cache.js (hunk 3 above)
Comment (sanjaypojo) [inline worker/image-cache.js]: Yup, again, vestigial code. Had initially had items either pass a resolutionIndex or a static photoID. Now this isn't required.

**Pull request merged**